server/cmwell-it/src/it/scala/cmwell/it/AuthTests.scala

```diff
@@ -142,16 +143,18 @@ class AuthTests extends FunSpec with Matchers with Helpers with LazyLogging {
           cmw / "levels" / "r" / "infoton1",
           cmw / "levels" / "rw" / "infoton1",
           cmw / "tests" / "news" / "new-infoton-1"
-        ) foreach { path =>
-          val res = waitAndExtractBody(Http.post(path, exampleObj, Some("application/json"), headers = ("X-CM-Well-Type" -> "File") :: tokenHeader))
-          Json.parse(res) should be(jsonSuccess)
+        ) map { path =>
+          waitAndExtractBody(Http.post(path, exampleObj, Some("application/json"), headers = ("X-CM-Well-Type" -> "File") :: tokenHeader))
         }
-        indexingDuration.fromNow.block
+      }
+
+      it("should add some example data") {
+        exampleData.map(Json.parse(_) should be(jsonSuccess))
```

Comment:
very nice separation of the tests!
1. As a convention if the result is not needed you should use `foreach`
2. It's better to use the scalatest way. e.g. `all (exampleData.map(Json.parse)) should be(jsonSuccess)`